tmt-app/src/Components/MajorPage.jsx

```diff
@@ -29,6 +30,32 @@ function MajorPage() {
       >
         {data.name} (Major)
       </Typography>
+      <Grid container>
+        <Grid item>
+          <Box
+            sx={{
+              backgroundColor: "#F3E8DE",
```

Comment:
What theme color should this be? Can we pull from the theme hook instead of hard coding the color?

Reply:
I believe most of the colors used for this page are new and not in our theme yet. I will take the time to add them to our styling.